healthcheck_handler.py

```diff
@@ -114,6 +114,15 @@ def get_aws_credentials() -> dict:
     }
 
 
+def validate_aws_credentials(access_key=None, secret_key=None):
+    client_kwargs = {}
+    if access_key and secret_key:
+        client_kwargs["aws_access_key_id"] = access_key
+        client_kwargs["aws_secret_access_key"] = secret_key
+    sts = boto3.client("sts", **client_kwargs)
```

Comment:
i think here you will need to call the actual ECR function to validate if the access_key and secret_key can access ecr.
i maybe possible the access key and secret key is valid for sts but not for ECR